beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -387,6 +401,12 @@ impl SupportedProtocol {
                 ProtocolId::new(SupportedProtocol::BlobsByRangeV1, Encoding::SSZSnappy),
             ]);
         }
+        if fork_context.spec.peerdas_scheduled() {
+            supported.extend_from_slice(&[
+                ProtocolId::new(SupportedProtocol::DataColumnsByRootV1, Encoding::SSZSnappy),
+                ProtocolId::new(SupportedProtocol::DataColumnsByRangeV1, Encoding::SSZSnappy),
+            ]);
+        }
```

Comment:
This logic is the one modifying the advertising rules. Because PeerDAS is still a "soft fork" not tied to Electra this check will advertise the topics in this cases:

| EIP7594_FORK_EPOCH | topics advertised |
| - | - | 
| no value | no |
| 18446744073709551615 | no |
| 0 | yes |
| 128 | yes |